src/subcommands/logs.ts

```diff
@@ -0,0 +1,160 @@
+    token: rawArgs.token ? String(rawArgs.token) : null,
+    deployment: rawArgs.deployment ? String(rawArgs.deployment) : null,
+    project: rawArgs.project ? String(rawArgs.project) : null,
+  };
+
+  if (args.help) {
+    console.log(help);
+    Deno.exit(0);
+  }
+  const token = args.token ?? Deno.env.get("DENO_DEPLOY_TOKEN") ?? null;
+  if (token === null) {
+    console.error(help);
+    error("Missing access token. Set via --token or DENO_DEPLOY_TOKEN.");
+  }
+  if (args.project === null) {
+    console.error(help);
+    error("Missing project ID.");
+  }
+  if (rawArgs._.length > 1) {
+    console.error(help);
+    error("Too many positional arguments given.");
+  }
+
+  const opts = {
+    project: args.project,
+    prod: args.prod,
+    token,
+    deployment: args.deployment,
+  };
+
+  await logs(opts);
+}
+
+interface DeployOpts {
+  project: string;
+  deployment: string | null;
+  prod: boolean;
+  token: string;
+}
+
+async function logs(opts: DeployOpts): Promise<void> {
+  if (opts.prod && opts.deployment) {
+    error(
+      `You can't select a deployment and choose production flag at the same time`,
+    );
+  }
+  const projectSpinner = wait("Fetching project information...").start();
+  const api = API.fromToken(opts.token);
+  const project = await api.getProject(opts.project);
+  const projectDeployments = await api.getDeployments(opts.project);
+  if (project === null) {
+    projectSpinner.fail("Project not found.");
+    Deno.exit(1);
+  }
+  if (opts.prod) {
+    if (!project.hasProductionDeployment) {
+      projectSpinner.fail(`This project doesn't have a production deployment`);
+      Deno.exit(1);
+    }
+    opts.deployment = project.productionDeployment?.id || null;
+  }
+  if (projectDeployments === null) {
+    projectSpinner.fail("Project not found.");
+    Deno.exit(1);
+  }
+  projectSpinner.succeed(`Project: ${project?.name}`);
+  const logs = opts.deployment
+    ? api.getLogs(opts.project, opts.deployment)
+    : api.getLogs(opts.project, "latest");
+  if (logs === null) {
+    projectSpinner.fail("Project not found.");
+    Deno.exit(1);
+  }
+  try {
+    for await (const log of logs) {
+      if (typeof log == "string") {
+        console.log(log);
+        continue;
+      }
+      let color;
+      switch (log.level) {
+        case "debug": {
+          color = "grey";
+          break;
+        }
+        case "error": {
+          color = "red";
+          break;
+        }
+        case "info": {
+          color = "blue";
+          break;
+        }
+      }
+      if (log.message.startsWith("isolate start time")) {
+        console.log(
+          `%c${log.time}   %c${log.region}%c ${log.message.trim()}`,
+          "color: aquamarine",
+          "background-color: grey",
+          `color: ${color}`,
+        );
+      } else {
+        console.log(
+          `%c${log.time}   %c${log.message.trim()}`,
+          "color: aquamarine",
+          `color: ${color}`,
+        );
+      }
```

Comment:
```suggestion
        console.log(
          `%c${log.time}   %c${log.region}%c ${log.message.trim()}`,
          "color: aquamarine",
          "background-color: grey",
          `color: ${color}`,
        );
```
What's the intention behind this conditional?

Reply:
It was in order to replicate the behavior of the web version, where it only shows the region in the logs that starts with the string "isolate start time".

![image](https://user-images.githubusercontent.com/72663983/210625060-08cc5b52-25e4-4059-97a4-90a0aaf0a239.png)

Reply:
It's better to display the region as logs can be streamed from [many regions](https://deno.com/deploy/docs/regions).

Reply:
Re: the dashboard, we have an internal issue to improve it.